php/slim/public/index.php

```diff
@@ -5,8 +5,7 @@
 
 require_once __DIR__ . '/../vendor/autoload.php';
 
-
-$app = new App();
+$app = new App(['settings' => ['routerCacheFile' => '/tmp']]);
```

Comment:
I think this will break on IIS, no ?

Reply:
Yes, I always forget about Windows...This could be a call to `sys_get_temp_dir()` instead.

Reply:
could made this in an other `PR` ?
I'm updating results